### ISSUE TITLE
Default initializers in destructured declarations do not transpile

### ISSUE THREAD

**Issue by matAtWork** [MatAtBread/nodent#97]
eg:
```js
const {bar1 = async () => 1} = {};
console.log(bar1)
const bar2 = async () => 1;
console.log(bar2)
```

bar1 is incorrectly not transpiled.

The same issue occurs in default destructured function parameters.

**Comment by matAtWork** [comment]
@reconbot - FYI

**Comment by matAtWork** [comment]
Fixed by bb729ae878d86c682e13c6a69aca639a52159b24
Released as v3.1.2
